### PR TITLE
Dockerfile: Add LABEL maintainer

### PR DESCRIPTION
**Pull request by RaiVaibhav**
Closes  https://github.com/fossasia/open-event-orga-server/issues/4506

Changes: Replace `MAINTAINER` with `LABEL maintainer`

### CODE REVIEW

**Comment by RaiVaibhav** [comment]
@shubham-padia  @rahuldecoded @invinciblycool can you review it please

**Comment by RaiVaibhav** [comment]
@invinciblycool  does  PR is fine or i have to make  a changes and i think we can also include author name

**Comment by invinciblycool** [comment]
If you can include the author name, it will be well in good. Then @shubham-padia can approve and merge it.

**Comment by RaiVaibhav** [comment]
@invinciblycool  i tried to add name inside  but travis failing , i removed the name its works file ,i will check this problem later as currently i am in journey of 2 days from my college to my home,any suggestion will be helpful

**Comment by RaiVaibhav** [comment]
@invinciblycool @shubham-padia @rahuldecoded  can you review it please

**Comment by RaiVaibhav** [comment]
@invinciblycool your review also required regarding  in my `PR` as i have changed `psycopg2>=2.6.1` as i have read that it causes when `psycopg2 -2.7` is already thereso i maked it to `psycopg2>=2.6.1`  but in your latest [PR](https://github.com/fossasia/open-event-orga-server/pull/4513) you have maked it to `psycopg2==2.7.3.1` i don't know whether both works fine or I have to close a PR and wait till `Travis` problem is fixed

**Comment by invinciblycool** [comment]
@RaiVaibhav I have not understood clearly what you are trying to say but yes waiting till the Travis build issue is fixed would be a wise choice.

**Comment by RaiVaibhav** [comment]
@invinciblycool  i was saying changing it to directly to a version `2.7.3.1` will be a better choice or just using the versions which if is `psycopg2>=2.6.1` will be better??, i really don't know whcich will be better.I just read about the log of travis failing and googled it and after that i find that that this problem happen when when `2.7` version is already there

**Comment by RaiVaibhav** [comment]
@invinciblycool  and yes it working fine you can see my commit in this PR

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 FROM python:2-slim
-MAINTAINER Avi Aryan <avi.aryan123@gmail.com>
+LABEL maintainer="Avi Aryan <avi.aryan123@gmail.com>"
```
Comment (niranjan94) [inline Dockerfile]: Please change this to the fossasia email address or mine.

File: requirements/common.txt [hunk 2]
```diff
@@ -9,7 +9,7 @@ Flask-JWT>=0.3.2,<0.4
 requests-oauthlib>=0.7.0,<1
 icalendar>=3.11,<4
 requests[security]>=2.12.4,<3
-psycopg2>=2.6.2,<2.7
+psycopg2>=2.6.1
```
Comment (niranjan94) [inline requirements/common.txt]: Please explain the reason for this change. `<2.7` was intentional as we did not want a breaking update.

File: Dockerfile (hunk 1 above)
Comment (RaiVaibhav) [reply Dockerfile]: 👍

File: requirements/common.txt (hunk 2 above)
Comment (RaiVaibhav) [reply requirements/common.txt]: @niranjan94  i google the log ,why travis failing so i find out this happen when version 2.7 is already there so i removed the <2.7 ,and added 2.6.1 so the version which is after 2.6.1 will get added automatically but i forgot that removing upper limit can cause `breaking update ` problem, I will wait till Travis problem gets fixed and will open a new PR will add your name and email address

**Comment by niranjan94** [comment]
@RaiVaibhav in future, please don't close and open new PRs for the same issue. Always update existing PR.

**Comment by RaiVaibhav** [comment]
👍

**Pull request closed without merging**